song_search/songs/views.py

```diff
@@ -1,26 +1,35 @@
 from django.http import HttpResponseRedirect
 from django.shortcuts import render
-
+from models import Songs
 
 def index(request):
     return render(request, 'index.html')
 
 
 def search_songs(request):
     search_text = (request.GET['search_box']).strip()
-    search_filter = request.GET.has_key('filter')
-    if len(search_text) > 0 and search_filter:
-        context = {
-            'songs': [{
-                'name': 'Dancing Queen',
-                'artist': 'ABBA',
-                'album': 'ABBA',
-                'lyrics': 'You are the dancing queen',
+    has_search_filter = request.GET.has_key('filter')
+    if len(search_text) > 0 and has_search_filter:
+        results_list = []
+        selected_filter = request.GET['filter']
+        if selected_filter == 'artist':
+            filt = Songs.objects.filter(artist__icontains = search_text)
+        elif selected_filter == 'song_name':
+            filt = Songs.objects.filter(song_name__icontains = search_text)
+        else:
+            filt = Songs.objects.filter(lyrics__icontains = search_text)
+        for row in filt:
```

Comment:
funcionaria melhor se voce alterasse para `for row in filt.order_by('song_name')` ?
Qual diferenca faria no resultado?